cmd/thor/utils.go

```diff
@@ -754,3 +754,16 @@ func readIntFromUInt64Flag(val uint64) (int, error) {
 
 	return i, nil
 }
+
+func parseTracerList(list string) map[string]interface{} {
```

Comment:
Should we validate inputs? As in if I start thor with `thor --allowed-tracers="darren,123"`

Reply:
If someone tries to use the `123` tracer it will fail because it's not loaded. I think it should be ok ?
We could have a list of available tracers, but I didn't want to change too much the existing debug package.

Reply:
but what about darren tracer ? I think maybe it is worth validating it

Reply:
I believe that if the node operator is specifying a tracer by name, he/she should double-check that the name is correct. Happy to iterate on this feature in the future if there's feedback in that direction.